elements/pfe-primary-detail/test/pfe-primary-detail_test.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <script src="/components/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src="/components/web-component-tester/browser.js"></script>
+  <script type="module" src="../dist/pfe-primary-detail.js"></script>
+</head>
+
+<body>
+  <pfe-primary-detail>
+    <!-- Default slot -->
+    <h2>This is pfe-primary-detail</h2>
+    
+  </pfe-primary-detail>
```

Comment:
Needs tests written

Reply:
Yep! Wanted to see how I did on everything else first. Also need to get storybook working.